### DISCUSSION TITLE
Is it possible to Add a Contrast Setting ?

### DISCUSSION THREAD

**Discussion by ceut2k4** [Ralim/IronOS#836] [Ideas]
Hello everyone !
I follow and use the Ralim FW since the 2.06 release, and I can say that it has really change my TS-100 into a super great tool, so thank you for all the work you do on it !

I have only a single request : as I find the Oled screen really too bright for my eyes, is there a way to add a contrast menu on this new IronOS ?
I have checked the M00881 datasheet, and it seems that with sending 0x81 on the I2C bus, we can access to the contrast setting (datasheet page 14 "Write_Command(0x81); //Set Contrast Control" )
But I can't find any other information on the contrast value or other thing.

Thank you very much !

**Comment by Ralim** [comment]
Hello 👋🏼 

Indeed currently this firmware runs the screen at the maximum duty cycle / brightness.
This could certainly be added as a menu option if people would like it.

**Reply by ceut2k4** [reply]
Thank you for your fast reply !

So hope that people will like to have this setting in the IronOS menu :)

Also, I think that it might help with Oled screen wearing... (and eyes saving)

**Reply by discip** [reply]
@ceut2k4
@Ralim
Good news,
@alvinhochun created this [PR](https://github.com/Ralim/IronOS/pull/1010#issue-689700822), which covers exactly what @ceut2k4 initially asked for, but someone would have to make it persistent! 😁

regards

**Reply by discip** [reply]
@Ralim
Implemented this already! 👍
